skatepark/inc/patterns/columns-in-container.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Columns in container.
+ *
+ * @package Skatepark
+ */
+
+return array(
+	'title'      => __( 'Columns in container', 'skatepark' ),
+	'categories' => array( 'skatepark' ),
+	'content'    => '<!-- wp:spacer {"height":40} -->
+	<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
+ 
+	<!-- wp:group {"align":"wide","style":{"border":{"width":"3px","style":"solid"},"spacing":{"padding":{"top":"3%","right":"4%","bottom":"4%","left":"4%"}}},"borderColor":"primary","className":"columns-in-container","layout":{"inherit":true}} -->
+	<div class="wp-block-group alignwide columns-in-container has-border-color has-primary-border-color" style="border-style:solid;border-width:3px;padding-top:3%;padding-right:4%;padding-bottom:4%;padding-left:4%"><!-- wp:columns {"align":"wide"} -->
```

Comment:
I think when https://github.com/Automattic/themes/pull/4436 merges we can make the vertical and horizontal paddings the same.